core/lib/dependency-graph/network-node.js

```diff
@@ -26,42 +29,49 @@ class NetworkNode extends BaseNode {
    * @return {number}
    */
   get startTime() {
-    return this._record.rendererStartTime * 1000;
+    return this._request.rendererStartTime * 1000;
   }
 
   /**
    * @return {number}
    */
   get endTime() {
-    return this._record.networkEndTime * 1000;
+    return this._request.networkEndTime * 1000;
   }
 
   /**
-   * @return {LH.Artifacts.NetworkRequest}
+   * @return {T}
    */
   get record() {
-    return this._record;
+    return /** @type {T} */ (this._request.record);
```

Comment:
nit: I wouldn't expect this type cast to be necessary, are there errors if it's removed?